creator/views.py

```diff
@@ -0,0 +1,31 @@
+from graphene_file_upload.django import FileUploadGraphQLView
+
+
+class SentryGraphQLView(FileUploadGraphQLView):
+    """
+    Sets the Sentry transaction name to be that of the GraphQL operation.
+    This helps break down all the /graphql requests into more actionable
+    buckets within Sentry.
+    """
+
+    def execute_graphql_request(self, *args, **kwargs):
+        """
+        Attempt to extract the operation name and use it to set the Sentry
+        transaction name.
+        """
+
+        # args[4] is the operationName, see super's signature:
+        # https://github.com/graphql-python/graphene-django/blob/main/graphene_django/views.py#L278-L280
+        if (
+            args
+            and len(args) >= 5
+            and args[4] is not None
+            and len(args[4]) > 0
+        ):
+            from sentry_sdk import Hub
+
+            transaction = Hub.current.scope.transaction
```

Comment:
When will transaction be None?

Reply:
Also just saw this in the docs. It looks like its doing the same thing but with the context manager: https://docs.sentry.io/platforms/python/guides/django/enriching-events/transaction-name/. Do you want to use that?

Reply:
The type says it's optional so I'm accounting for that: https://github.com/getsentry/sentry-python/blob/7ba60bda29d671bbef79ae5646fb062c898efc6a/sentry_sdk/scope.py#L146

Yeah I tried the context manager but there was something preventing it from working.